Sources/Dictionary/dictionaryManager.swift

```diff
@@ -0,0 +1,19 @@
+import OS
+
+public class DictionaryManager {
+	let gameWords = words
+
+	public init() {
+	}
+
+	public func getRandomIndex() -> Int {
+		OS.srand(UInt32(time(nil)))
+		let randNum = Int(OS.rand())
+		return randNum % gameWords.count
+	}
+
+	public func getRandomWord() -> String {
+		let index = self.getRandomIndex()
```

Comment:
I think the self is superfluous here. Check out the github's [swift style guide](https://github.com/github/swift-style-guide#only-explicitly-refer-to-self-when-required)